member.py

```diff
@@ -190,6 +189,7 @@ def get_add_gig_band_list(cls, req, the_member_key):
 
     @classmethod
     def get_forums(cls, req, the_member_key):
+        import forum
```

Comment:
The forum stuff is pretty half-baked. We just comment out anything giving us trouble.